wiki/Community/Projects/fr.md

```diff
@@ -1,5 +1,6 @@
 ---
-outdated: true
```

Comment:
this alone should be enough for the outdated article checker to keep quiet. will take a look at what's going on under the hood

Reply:
> if the translation was already outdated before, then we just don't put in `outdated_since`
> 
> (https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#outdated-articles)

so should I just remove `outdated_since` and that's it?

Reply:
no, add back `outdated: true`